pkg/policy/rule.go

```diff
@@ -285,6 +270,8 @@ func (r *rule) resolveL4Policy(ctx *SearchContext, state *traceState, result *L4
 	return nil, nil
 }
 
+// ********************** CIDR POLICY **********************
```

Comment:
I think this is kind of change is suggesting we should break this down into separate files, but I won't press you to do this in this PR.